src/trans/codegen_c.cpp

```diff
@@ -1210,7 +1210,9 @@ namespace {
                     {
                         args.push_back(c);
                     }
+#if !defined(__APPLE__)
```

Comment:
This should probably get the target somehow rather than defining this based on the host.

Reply:
@bjorn3 probably, anyway, it seems as easy way to do.

Also here: https://github.com/thepowersgang/mrustc/issues/138 you may read near the same discussion.

Reply:
BTW: used Compiler is hardcoded inside the code :)

Reply:
Go ahead and remove this part of the changeset, I'm making it a part of `m_linker_opts`

Reply:
@thepowersgang here it is.